### DISCUSSION TITLE
System request

### DISCUSSION THREAD

**Discussion by WuodMildura123** [community/community#132468] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi people, Which mobile app is more efficient in marking items so that if they are taken or stolen we can be notified

**Comment by NugrohoRahmanto** [comment]
if you have posted on internet, there's no way you notice that its stolen. but you can added watermark to your design

**Reply by WuodMildura123** [reply]
What you mean?

**Comment by WuodMildura123** [comment]
What's the name of that device used by mortuary attendees
